charts/wa-proto-frontend/requirements.yaml

```diff
@@ -0,0 +1,9 @@
+dependencies:
+  - name: nodejs
+    version: ~1.8.0
+    repository: '@hmctspublic'
+  - name: redis
+    version: "10.0.0"
+    repository: "https://charts.bitnami.com/bitnami"
+    tags:
```

Comment:
I usually prefer conditions than tags. Example: https://github.com/hmcts/chart-java/blob/master/java/Chart.yaml#L24

Also, u need to add this to Chart.yaml. you are duplicating nodejs chart here. (that's how v2 works)

Reply:
done